### ISSUE TITLE
Parser Bug in Release 2018-04: OPERATE Keyword

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-common#481]
The following works in the commercial simulator:

```
-- Poro-perm transform 
OPERATE
WORK1 6* 'MULTX' PORO 1.0 /
WORK2 6* 'MINLIM' WORK1 0.01 /
PERMX 6* 'MULTP' WORK2 3.2627058E+07 8.2569 /
/

```

But bad things happen in Flow:

```
Failed to create valid EclipseState object.
Exception caught: Fatal error processing OPERATE keyword - invalid/undefined keyword: WORK1
terminate called after throwing an instance of 'std::invalid_argument'
  what():  Fatal error processing OPERATE keyword - invalid/undefined keyword: WORK1
```

**Comment by joakim-hove** [comment]
OK - I have now fine-read the section in the manual about  `"WORK*` arrays - no that is currently not supported. The specification of work arrays seems to very tightly bound to the Eclipse simulation; I would say it is a medium sized task. Not clear to me if this should be prioritised.

**Comment by OPMUSER** [comment]
Yes I agree, I think we should just trap this and issue a warning message instead, "Unknown Property Array in OPERATE Keyword - Keyword Skipped", or something similar.

Using OPERATE and the ACTIONX keywords is very cumbersome and error prone as the process is forced to use the existing keyword structure. I would suggest that instead that we use OCTAVE (similar to ResInsight) instead,. That way we have greater functionality and flexibility, but I'm sure implementing this is much more complex.

Just a suggestion.

**Comment by andlaus** [comment]
didn't you propose a PYTHON keyword at the developer meeting in April, @joakim-hove?

**Comment by joakim-hove** [comment]
Yes

**Comment by OPMUSER** [comment]
ResInsight users Octave, so I thought that would be a could fit, but Python would work also, anything is better than the ACTION set of keywords :-)
